samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/ManageController.cs

```diff
@@ -451,5 +451,42 @@ public async Task<ActionResult> SelectRecoveryAuthenticatorAsync(SelectRecoveryA
             }
         }
 
+        [HttpPost]
+        [AllowAnonymous]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> ChangePasswordRecoveryTokenAsync(ChangePasswordRecoveryTokenViewModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View("ChangePasswordRecoveryToken", model);
+            }
+
+            try
+            {
+                var recoverPasswordResponse = await _idxClient.RecoverPasswordAsync(
```

Comment:
The method name `ChangePasswordRecoveryTokenAsync` feels confusing to me.  It tells me that I'm changing the `recovery token` which is of type `password`; while this line `... _idxClient.RecoverPassword...` implies that I'm changing password using a recovery token.  How about `ChangePasswordWithRecoveryTokenAsync` or even `RecoverPasswordAsync` as the controller method name?